labgrid/driver/fastbootdriver.py

```diff
@@ -91,9 +92,10 @@ def getvar(self, var):
             raise NotImplementedError('Retrieving a list of all variables is not supported yet')
 
         cmd = ['getvar', var]
-        proc = subprocess.run(self._get_fastboot_prefix() + cmd, stdout=subprocess.PIPE,
-                              stderr=subprocess.STDOUT, check=True, universal_newlines=True)
-        values = AndroidFastbootDriver._filter_fastboot_output(proc.stdout, '{}: '.format(var))
+        output = processwrapper.check_output(self._get_fastboot_prefix() + cmd)
+        values = AndroidFastbootDriver._filter_fastboot_output(
```

Comment:
Is _filter_fastboot_output able to handle the additional lines from stderr?

Reply:
I don't have a device to test this at the moment.